src/Integration/Binding/DotNetBindingConfigProvider.cs

```diff
@@ -96,8 +96,11 @@ public async Task<IBindingConfigFile> GetConfigurationAsync(SonarQubeQualityProf
                 return null;
             }
 
+            // duncanp - WIP - next line will be removed when the RoslynProfileExporter is removed
+            var nugetPackages = roslynProfileExporter.Deployment.NuGetPackages.Select(x => new Core.CSharpVB.NuGetPackageInfo(x.Id, x.Version));
```

Comment:
Temporary workaround to limit the size of this PR.
After this I'll update the `DotNetBindingConfigProvider` to stop using the `RoslynProfileExporter` entirely.